yjit.rb

```diff
@@ -318,6 +318,7 @@ def _print_stats(out: $stderr) # :nodoc:
       out.puts "compilation_failure:   " + format_number(13, compilation_failure) if compilation_failure != 0
       out.puts "compiled_iseq_entry:   " + format_number(13, stats[:compiled_iseq_entry])
       out.puts "cold_iseq_entry:       " + format_number_pct(13, stats[:cold_iseq_entry], stats[:compiled_iseq_entry] + stats[:cold_iseq_entry])
+      out.puts "live_iseq_count:       " + format_number(13, stats[:live_iseq_count])
```

Comment:
```suggestion
      out.puts "live_iseq_count:       " + format_number(13, stats[:live_iseq_count])
      out.puts "compiled_iseq_entry:   " + format_number(13, stats[:compiled_iseq_entry])
      out.puts "cold_iseq_entry:       " + format_number_pct(13, stats[:cold_iseq_entry], stats[:compiled_iseq_entry] + stats[:cold_iseq_entry])
```

I'm tempted to put `live_iseq_count` first since it would follow the "live ISEQ" -> "compiled ISEQ" or "cold ISEQ" ->  "compiled block ID" -> "compiled block" or "compiled branch" lifecycle of these objects.